big_o/test/test_big_o.py

```diff
@@ -105,8 +120,7 @@ def test_big_o(self):
             (dummy_constant_function, compl.Constant, (1000, 10000)),
             (dummy_linear_function, compl.Linear, (100, 5000)),
             (dummy_quadratic_function, compl.Quadratic, (1, 100)),
-            (lambda n: np.sort(random_array[:n], kind='heapsort'),
-             compl.Linearithmic, (100, random_array.shape[0])),
+            (dummy_linearithmic_function, compl.Linearithmic, (10, 5000)),
```

Comment:
with this change, you can set `n_repeats=1` below, it's going to be much faster!

Reply:
Updated :)